### PR TITLE
Manage a credit org microlesson added

### PR DESCRIPTION
**Pull request by JV0812**
## Purpose of this pull request

This pull request is to add the microlesson details to the Update child org section.

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

## Ticket (if applicable)

[Add a link to micro lesson video 'Managing orgs with Flexible Credit Allocation'](https://sumologic.atlassian.net/browse/DOCS-331)

### CODE REVIEW

**Review by jpipkin1 — APPROVED** [review]
Looks good, @JV0812. I had to make one correction (see commit "[Add import Iframe tag](https://github.com/SumoLogic/sumologic-documentation/pull/4307/commits/70d6c7ee332154bbdf6dcd1c0a1f1899551a3037)").

File: docs/manage/manage-subscription/create-manage-orgs.md [hunk 1]
```diff
@@ -109,6 +109,20 @@ After you create a new org, you can’t delete it.
 
 ## Update an org's credits allocation
 
+:::sumo Micro Lesson
+Manage a Child Org
```
Comment (kimsauce) [inline docs/manage/manage-subscription/create-manage-orgs.md:113]:
```suggestion
Watch this micro lesson to learn more about managing a child org.
```

File: docs/manage/manage-subscription/create-manage-orgs-service-providers.md [hunk 2]
```diff
@@ -160,6 +161,20 @@ If a POV trial org is not upgraded to Enterprise plan after 45 days, the org wil
 
 ## Update an org's credits allocation
 
+:::sumo Micro Lesson
+Manage a Child Org
```
Comment (kimsauce) [inline docs/manage/manage-subscription/create-manage-orgs-service-providers.md:165]:
```suggestion
Watch this micro lesson to learn more about managing a child org.
```

**Review by kimsauce — APPROVED** [review]
Updated the micro lesson admonition wording to align with its counterparts (i.e., _Watch this micro lesson to learn more about ..._)

**Comment by JV0812** [comment]
Thanks John and Kim for your review and suggestion.

**Pull request merged**